[receiver/sqlserver] Broken Windows unit test: TestQueryTextAndPlanQueryContents

### Component(s)

receiver/sqlserver

### Describe the issue you're reporting

[Failing CI/CD action](https://github.com/open-telemetry/opentelemetry-collector-contrib/actions/runs/13957569151/job/39072317433#step:8:863)

Failure output:
```
=== FAIL: . TestQueryTextAndPlanQueryContents/Test_query_text_and_query_plan_without_instance_name (0.00s)
    queries_test.go:109: 
        	Error Trace:	D:/a/opentelemetry-collector-contrib/opentelemetry-collector-contrib/receiver/sqlserverreceiver/queries_test.go:109
        	Error:      	Not equal: 
        	            	expected: "DECLARE @lookbackTime INT = -60;\nDECLARE @topNValue INT = 1000;\nwith qstats as (\n\tSELECT TOP(@topNValue)\n\t\tREPLACE(@@SERVERNAME,'\\',':') AS [sql_instance],\n\t\tHOST_NAME() AS [computer_name],\n\t\tMAX(qs.plan_handle) AS query_plan_handle,\n\t\tqs.query_hash AS query_hash,\n\t\tqs.query_plan_hash AS query_plan_hash,\n\t\tSUM(qs.execution_count) AS execution_count,\n\t\tSUM(qs.total_elapsed_time) AS total_elapsed_time,\n\t\tSUM(qs.total_worker_time) AS total_worker_time,\n\t\tSUM(qs.total_logical_reads) AS total_logical_reads,\n\t\tSUM(qs.total_physical_reads) AS total_physical_reads,\n\t\tSUM(qs.total_logical_writes) AS total_logical_writes,\n\t\tSUM(qs.total_rows) AS total_rows,\n\t\tSUM(qs.total_grant_kb) as total_grant_kb\n\tFROM sys.dm_exec_query_stats AS qs\n\tWHERE qs.last_execution_time BETWEEN DATEADD(SECOND, @lookbackTime, GETDATE()) AND GETDATE() \n\tGROUP BY\n\t\tqs.query_hash,\n\t\tqs.query_plan_hash\n)\nSELECT qs.*,\n\tSUBSTRING(st.text, (stats.statement_start_offset / 2) + 1,\n\t\t\t ((CASE statement_end_offset\n\t\t\t\t   WHEN -1 THEN DATALENGTH(st.text)\n\t\t\t\t   ELSE stats.statement_end_offset END - stats.statement_start_offset) / 2) + 1) AS query_text,\n\tISNULL(qp.query_plan, '') AS query_plan\nFROM qstats AS qs\n\t\tINNER JOIN sys.dm_exec_query_stats AS stats on qs.query_plan_handle = stats.plan_handle\n\t\tCROSS APPLY sys.dm_exec_query_plan(qs.query_plan_handle) AS qp\n\t\tCROSS APPLY sys.dm_exec_sql_text(qs.query_plan_handle) AS st;"
make[2]: *** [../../Makefile.Common:101: test] Error 1
        	            	actual  : "DECLARE @lookbackTime INT = -60;\r\nDECLARE @topNValue INT = 1000;\r\nwith qstats as (\r\n\tSELECT TOP(@topNValue)\r\n\t\tREPLACE(@@SERVERNAME,'\\',':') AS [sql_instance],\r\n\t\tHOST_NAME() AS [computer_name],\r\n\t\tMAX(qs.plan_handle) AS query_plan_handle,\r\n\t\tqs.query_hash AS query_hash,\r\n\t\tqs.query_plan_hash AS query_plan_hash,\r\n\t\tSUM(qs.execution_count) AS execution_count,\r\n\t\tSUM(qs.total_elapsed_time) AS total_elapsed_time,\r\n\t\tSUM(qs.total_worker_time) AS total_worker_time,\r\n\t\tSUM(qs.total_logical_reads) AS total_logical_reads,\r\n\t\tSUM(qs.total_physical_reads) AS total_physical_reads,\r\n\t\tSUM(qs.total_logical_writes) AS total_logical_writes,\r\n\t\tSUM(qs.total_rows) AS total_rows,\r\n\t\tSUM(qs.total_grant_kb) as total_grant_kb\r\n\tFROM sys.dm_exec_query_stats AS qs\r\n\tWHERE qs.last_execution_time BETWEEN DATEADD(SECOND, @lookbackTime, GETDATE()) AND GETDATE() \r\n\tGROUP BY\r\n\t\tqs.query_hash,\r\n\t\tqs.query_plan_hash\r\n)\r\nSELECT qs.*,\r\n\tSUBSTRING(st.text, (stats.statement_start_offset / 2) + 1,\r\n\t\t\t ((CASE statement_end_offset\r\n\t\t\t\t   WHEN -1 THEN DATALENGTH(st.text)\r\n\t\t\t\t   ELSE stats.statement_end_offset END - stats.statement_start_offset) / 2) + 1) AS query_text,\r\n\tISNULL(qp.query_plan, '') AS query_plan\r\nFROM qstats AS qs\r\n\t\tINNER JOIN sys.dm_exec_query_stats AS stats on qs.query_plan_handle = stats.plan_handle\r\n\t\tCROSS APPLY sys.dm_exec_query_plan(qs.query_plan_handle) AS qp\r\n\t\tCROSS APPLY sys.dm_exec_sql_text(qs.query_plan_handle) AS st;"
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1,33 +1,33 @@
        	            	-DECLARE @lookbackTime INT = -60;
        	            	-DECLARE @topNValue INT = 1000;
        	            	-with qstats as (
        	            	-	SELECT TOP(@topNValue)
        	            	-		REPLACE(@@SERVERNAME,'\',':') AS [sql_instance],
        	            	-		HOST_NAME() AS [computer_name],
        	            	-		MAX(qs.plan_handle) AS query_plan_handle,
        	            	-		qs.query_hash AS query_hash,
        	            	-		qs.query_plan_hash AS query_plan_hash,
        	            	-		SUM(qs.execution_count) AS execution_count,
        	            	-		SUM(qs.total_elapsed_time) AS total_elapsed_time,
        	            	-		SUM(qs.total_worker_time) AS total_worker_time,
        	            	-		SUM(qs.total_logical_reads) AS total_logical_reads,
        	            	-		SUM(qs.total_physical_reads) AS total_physical_reads,
        	            	-		SUM(qs.total_logical_writes) AS total_logical_writes,
        	            	-		SUM(qs.total_rows) AS total_rows,
        	            	-		SUM(qs.total_grant_kb) as total_grant_kb
        	            	-	FROM sys.dm_exec_query_stats AS qs
        	            	-	WHERE qs.last_execution_time BETWEEN DATEADD(SECOND, @lookbackTime, GETDATE()) AND GETDATE() 
        	            	-	GROUP BY
        	            	-		qs.query_hash,
        	            	-		qs.query_plan_hash
        	            	-)
        	            	-SELECT qs.*,
        	            	-	SUBSTRING(st.text, (stats.statement_start_offset / 2) + 1,
        	            	-			 ((CASE statement_end_offset
        	            	-				   WHEN -1 THEN DATALENGTH(st.text)
        	            	-				   ELSE stats.statement_end_offset END - stats.statement_start_offset) / 2) + 1) AS query_text,
        	            	-	ISNULL(qp.query_plan, '') AS query_plan
        	            	-FROM qstats AS qs
        	            	-		INNER JOIN sys.dm_exec_query_stats AS stats on qs.query_plan_handle = stats.plan_handle
        	            	-		CROSS APPLY sys.dm_exec_query_plan(qs.query_plan_handle) AS qp
        	            	+DECLARE @lookbackTime INT = -60;
        	            	+DECLARE @topNValue INT = 1000;
        	            	+with qstats as (
        	            	+	SELECT TOP(@topNValue)
        	            	+		REPLACE(@@SERVERNAME,'\',':') AS [sql_instance],
        	            	+		HOST_NAME() AS [computer_name],
        	            	+		MAX(qs.plan_handle) AS query_plan_handle,
        	            	+		qs.query_hash AS query_hash,
        	            	+		qs.query_plan_hash AS query_plan_hash,
        	            	+		SUM(qs.execution_count) AS execution_count,
        	            	+		SUM(qs.total_elapsed_time) AS total_elapsed_time,
        	            	+		SUM(qs.total_worker_time) AS total_worker_time,
        	            	+		SUM(qs.total_logical_reads) AS total_logical_reads,
        	            	+		SUM(qs.total_physical_reads) AS total_physical_reads,
        	            	+		SUM(qs.total_logical_writes) AS total_logical_writes,
        	            	+		SUM(qs.total_rows) AS total_rows,
        	            	+		SUM(qs.total_grant_kb) as total_grant_kb
        	            	+	FROM sys.dm_exec_query_stats AS qs
        	            	+	WHERE qs.last_execution_time BETWEEN DATEADD(SECOND, @lookbackTime, GETDATE()) AND GETDATE() 
        	            	+	GROUP BY
        	            	+		qs.query_hash,
        	            	+		qs.query_plan_hash
        	            	+)
        	            	+SELECT qs.*,
        	            	+	SUBSTRING(st.text, (stats.statement_start_offset / 2) + 1,
        	            	+			 ((CASE statement_end_offset
        	            	+				   WHEN -1 THEN DATALENGTH(st.text)
        	            	+				   ELSE stats.statement_end_offset END - stats.statement_start_offset) / 2) + 1) AS query_text,
        	            	+	ISNULL(qp.query_plan, '') AS query_plan
        	            	+FROM qstats AS qs
        	            	+		INNER JOIN sys.dm_exec_query_stats AS stats on qs.query_plan_handle = stats.plan_handle
        	            	+		CROSS APPLY sys.dm_exec_query_plan(qs.query_plan_handle) AS qp
        	            	 		CROSS APPLY sys.dm_exec_sql_text(qs.query_plan_handle) AS st;
        	Test:       	TestQueryTextAndPlanQueryContents/Test_query_text_and_query_plan_without_instance_name

=== FAIL: . TestQueryTextAndPlanQueryContents/Test_query_text_and_query_plan_with_instance_name (0.00s)
    queries_test.go:109: 
        	Error Trace:	D:/a/opentelemetry-collector-contrib/opentelemetry-collector-contrib/receiver/sqlserverreceiver/queries_test.go:109
        	Error:      	Not equal: 
        	            	expected: "DECLARE @lookbackTime INT = -120;\nDECLARE @topNValue INT = 2000;\nwith qstats as (\n\tSELECT TOP(@topNValue)\n\t\tREPLACE(@@SERVERNAME,'\\',':') AS [sql_instance],\n\t\tHOST_NAME() AS [computer_name],\n\t\tMAX(qs.plan_handle) AS query_plan_handle,\n\t\tqs.query_hash AS query_hash,\n\t\tqs.query_plan_hash AS query_plan_hash,\n\t\tSUM(qs.execution_count) AS execution_count,\n\t\tSUM(qs.total_elapsed_time) AS total_elapsed_time,\n\t\tSUM(qs.total_worker_time) AS total_worker_time,\n\t\tSUM(qs.total_logical_reads) AS total_logical_reads,\n\t\tSUM(qs.total_physical_reads) AS total_physical_reads,\n\t\tSUM(qs.total_logical_writes) AS total_logical_writes,\n\t\tSUM(qs.total_rows) AS total_rows,\n\t\tSUM(qs.total_grant_kb) as total_grant_kb\n\tFROM sys.dm_exec_query_stats AS qs\n\tWHERE qs.last_execution_time BETWEEN DATEADD(SECOND, @lookbackTime, GETDATE()) AND GETDATE() AND @@SERVERNAME = 'instanceName'\n\tGROUP BY\n\t\tqs.query_hash,\n\t\tqs.query_plan_hash\n)\nSELECT qs.*,\n\tSUBSTRING(st.text, (stats.statement_start_offset / 2) + 1,\n\t\t\t ((CASE statement_end_offset\n\t\t\t\t   WHEN -1 THEN DATALENGTH(st.text)\n\t\t\t\t   ELSE stats.statement_end_offset END - stats.statement_start_offset) / 2) + 1) AS query_text,\n\tISNULL(qp.query_plan, '') AS query_plan\nFROM qstats AS qs\n\t\tINNER JOIN sys.dm_exec_query_stats AS stats on qs.query_plan_handle = stats.plan_handle\n\t\tCROSS APPLY sys.dm_exec_query_plan(qs.query_plan_handle) AS qp\n\t\tCROSS APPLY sys.dm_exec_sql_text(qs.query_plan_handle) AS st;"
        	            	actual  : "DECLARE @lookbackTime INT = -120;\r\nDECLARE @topNValue INT = 2000;\r\nwith qstats as (\r\n\tSELECT TOP(@topNValue)\r\n\t\tREPLACE(@@SERVERNAME,'\\',':') AS [sql_instance],\r\n\t\tHOST_NAME() AS [computer_name],\r\n\t\tMAX(qs.plan_handle) AS query_plan_handle,\r\n\t\tqs.query_hash AS query_hash,\r\n\t\tqs.query_plan_hash AS query_plan_hash,\r\n\t\tSUM(qs.execution_count) AS execution_count,\r\n\t\tSUM(qs.total_elapsed_time) AS total_elapsed_time,\r\n\t\tSUM(qs.total_worker_time) AS total_worker_time,\r\n\t\tSUM(qs.total_logical_reads) AS total_logical_reads,\r\n\t\tSUM(qs.total_physical_reads) AS total_physical_reads,\r\n\t\tSUM(qs.total_logical_writes) AS total_logical_writes,\r\n\t\tSUM(qs.total_rows) AS total_rows,\r\n\t\tSUM(qs.total_grant_kb) as total_grant_kb\r\n\tFROM sys.dm_exec_query_stats AS qs\r\n\tWHERE qs.last_execution_time BETWEEN DATEADD(SECOND, @lookbackTime, GETDATE()) AND GETDATE() AND @@SERVERNAME = 'instanceName'\r\n\tGROUP BY\r\n\t\tqs.query_hash,\r\n\t\tqs.query_plan_hash\r\n)\r\nSELECT qs.*,\r\n\tSUBSTRING(st.text, (stats.statement_start_offset / 2) + 1,\r\n\t\t\t ((CASE statement_end_offset\r\n\t\t\t\t   WHEN -1 THEN DATALENGTH(st.text)\r\n\t\t\t\t   ELSE stats.statement_end_offset END - stats.statement_start_offset) / 2) + 1) AS query_text,\r\n\tISNULL(qp.query_plan, '') AS query_plan\r\nFROM qstats AS qs\r\n\t\tINNER JOIN sys.dm_exec_query_stats AS stats on qs.query_plan_handle = stats.plan_handle\r\n\t\tCROSS APPLY sys.dm_exec_query_plan(qs.query_plan_handle) AS qp\r\n\t\tCROSS APPLY sys.dm_exec_sql_text(qs.query_plan_handle) AS st;"
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1,33 +1,33 @@
        	            	-DECLARE @lookbackTime INT = -120;
        	            	-DECLARE @topNValue INT = 2000;
        	            	-with qstats as (
        	            	-	SELECT TOP(@topNValue)
        	            	-		REPLACE(@@SERVERNAME,'\',':') AS [sql_instance],
        	            	-		HOST_NAME() AS [computer_name],
        	            	-		MAX(qs.plan_handle) AS query_plan_handle,
        	            	-		qs.query_hash AS query_hash,
        	            	-		qs.query_plan_hash AS query_plan_hash,
        	            	-		SUM(qs.execution_count) AS execution_count,
        	            	-		SUM(qs.total_elapsed_time) AS total_elapsed_time,
        	            	-		SUM(qs.total_worker_time) AS total_worker_time,
        	            	-		SUM(qs.total_logical_reads) AS total_logical_reads,
        	            	-		SUM(qs.total_physical_reads) AS total_physical_reads,
        	            	-		SUM(qs.total_logical_writes) AS total_logical_writes,
        	            	-		SUM(qs.total_rows) AS total_rows,
        	            	-		SUM(qs.total_grant_kb) as total_grant_kb
        	            	-	FROM sys.dm_exec_query_stats AS qs
        	            	-	WHERE qs.last_execution_time BETWEEN DATEADD(SECOND, @lookbackTime, GETDATE()) AND GETDATE() AND @@SERVERNAME = 'instanceName'
        	            	-	GROUP BY
        	            	-		qs.query_hash,
        	            	-		qs.query_plan_hash
        	            	-)
        	            	-SELECT qs.*,
        	            	-	SUBSTRING(st.text, (stats.statement_start_offset / 2) + 1,
        	            	-			 ((CASE statement_end_offset
        	            	-				   WHEN -1 THEN DATALENGTH(st.text)
        	            	-				   ELSE stats.statement_end_offset END - stats.statement_start_offset) / 2) + 1) AS query_text,
        	            	-	ISNULL(qp.query_plan, '') AS query_plan
        	            	-FROM qstats AS qs
        	            	-		INNER JOIN sys.dm_exec_query_stats AS stats on qs.query_plan_handle = stats.plan_handle
        	            	-		CROSS APPLY sys.dm_exec_query_plan(qs.query_plan_handle) AS qp
        	            	+DECLARE @lookbackTime INT = -120;
        	            	+DECLARE @topNValue INT = 2000;
        	            	+with qstats as (
        	            	+	SELECT TOP(@topNValue)
        	            	+		REPLACE(@@SERVERNAME,'\',':') AS [sql_instance],
        	            	+		HOST_NAME() AS [computer_name],
        	            	+		MAX(qs.plan_handle) AS query_plan_handle,
        	            	+		qs.query_hash AS query_hash,
        	            	+		qs.query_plan_hash AS query_plan_hash,
        	            	+		SUM(qs.execution_count) AS execution_count,
        	            	+		SUM(qs.total_elapsed_time) AS total_elapsed_time,
        	            	+		SUM(qs.total_worker_time) AS total_worker_time,
        	            	+		SUM(qs.total_logical_reads) AS total_logical_reads,
        	            	+		SUM(qs.total_physical_reads) AS total_physical_reads,
        	            	+		SUM(qs.total_logical_writes) AS total_logical_writes,
        	            	+		SUM(qs.total_rows) AS total_rows,
        	            	+		SUM(qs.total_grant_kb) as total_grant_kb
        	            	+	FROM sys.dm_exec_query_stats AS qs
        	            	+	WHERE qs.last_execution_time BETWEEN DATEADD(SECOND, @lookbackTime, GETDATE()) AND GETDATE() AND @@SERVERNAME = 'instanceName'
        	            	+	GROUP BY
        	            	+		qs.query_hash,
        	            	+		qs.query_plan_hash
        	            	+)
        	            	+SELECT qs.*,
        	            	+	SUBSTRING(st.text, (stats.statement_start_offset / 2) + 1,
        	            	+			 ((CASE statement_end_offset
        	            	+				   WHEN -1 THEN DATALENGTH(st.text)
        	            	+				   ELSE stats.statement_end_offset END - stats.statement_start_offset) / 2) + 1) AS query_text,
        	            	+	ISNULL(qp.query_plan, '') AS query_plan
        	            	+FROM qstats AS qs
        	            	+		INNER JOIN sys.dm_exec_query_stats AS stats on qs.query_plan_handle = stats.plan_handle
        	            	+		CROSS APPLY sys.dm_exec_query_plan(qs.query_plan_handle) AS qp
        	            	 		CROSS APPLY sys.dm_exec_sql_text(qs.query_plan_handle) AS st;
        	Test:       	TestQueryTextAndPlanQueryContents/Test_query_text_and_query_plan_with_instance_name

=== FAIL: . TestQueryTextAndPlanQueryContents (0.00s)

=== FAIL: . TestQueryTextAndPlanQueryContents/Test_query_text_and_query_plan_without_instance_name (re-run 1) (0.00s)
    queries_test.go:109: 
        	Error Trace:	D:/a/opentelemetry-collector-contrib/opentelemetry-collector-contrib/receiver/sqlserverreceiver/queries_test.go:109
        	Error:      	Not equal: 
        	            	expected: "DECLARE @lookbackTime INT = -60;\nDECLARE @topNValue INT = 1000;\nwith qstats as (\n\tSELECT TOP(@topNValue)\n\t\tREPLACE(@@SERVERNAME,'\\',':') AS [sql_instance],\n\t\tHOST_NAME() AS [computer_name],\n\t\tMAX(qs.plan_handle) AS query_plan_handle,\n\t\tqs.query_hash AS query_hash,\n\t\tqs.query_plan_hash AS query_plan_hash,\n\t\tSUM(qs.execution_count) AS execution_count,\n\t\tSUM(qs.total_elapsed_time) AS total_elapsed_time,\n\t\tSUM(qs.total_worker_time) AS total_worker_time,\n\t\tSUM(qs.total_logical_reads) AS total_logical_reads,\n\t\tSUM(qs.total_physical_reads) AS total_physical_reads,\n\t\tSUM(qs.total_logical_writes) AS total_logical_writes,\n\t\tSUM(qs.total_rows) AS total_rows,\n\t\tSUM(qs.total_grant_kb) as total_grant_kb\n\tFROM sys.dm_exec_query_stats AS qs\n\tWHERE qs.last_execution_time BETWEEN DATEADD(SECOND, @lookbackTime, GETDATE()) AND GETDATE() \n\tGROUP BY\n\t\tqs.query_hash,\n\t\tqs.query_plan_hash\n)\nSELECT qs.*,\n\tSUBSTRING(st.text, (stats.statement_start_offset / 2) + 1,\n\t\t\t ((CASE statement_end_offset\n\t\t\t\t   WHEN -1 THEN DATALENGTH(st.text)\n\t\t\t\t   ELSE stats.statement_end_offset END - stats.statement_start_offset) / 2) + 1) AS query_text,\n\tISNULL(qp.query_plan, '') AS query_plan\nFROM qstats AS qs\n\t\tINNER JOIN sys.dm_exec_query_stats AS stats on qs.query_plan_handle = stats.plan_handle\n\t\tCROSS APPLY sys.dm_exec_query_plan(qs.query_plan_handle) AS qp\n\t\tCROSS APPLY sys.dm_exec_sql_text(qs.query_plan_handle) AS st;"
        	            	actual  : "DECLARE @lookbackTime INT = -60;\r\nDECLARE @topNValue INT = 1000;\r\nwith qstats as (\r\n\tSELECT TOP(@topNValue)\r\n\t\tREPLACE(@@SERVERNAME,'\\',':') AS [sql_instance],\r\n\t\tHOST_NAME() AS [computer_name],\r\n\t\tMAX(qs.plan_handle) AS query_plan_handle,\r\n\t\tqs.query_hash AS query_hash,\r\n\t\tqs.query_plan_hash AS query_plan_hash,\r\n\t\tSUM(qs.execution_count) AS execution_count,\r\n\t\tSUM(qs.total_elapsed_time) AS total_elapsed_time,\r\n\t\tSUM(qs.total_worker_time) AS total_worker_time,\r\n\t\tSUM(qs.total_logical_reads) AS total_logical_reads,\r\n\t\tSUM(qs.total_physical_reads) AS total_physical_reads,\r\n\t\tSUM(qs.total_logical_writes) AS total_logical_writes,\r\n\t\tSUM(qs.total_rows) AS total_rows,\r\n\t\tSUM(qs.total_grant_kb) as total_grant_kb\r\n\tFROM sys.dm_exec_query_stats AS qs\r\n\tWHERE qs.last_execution_time BETWEEN DATEADD(SECOND, @lookbackTime, GETDATE()) AND GETDATE() \r\n\tGROUP BY\r\n\t\tqs.query_hash,\r\n\t\tqs.query_plan_hash\r\n)\r\nSELECT qs.*,\r\n\tSUBSTRING(st.text, (stats.statement_start_offset / 2) + 1,\r\n\t\t\t ((CASE statement_end_offset\r\n\t\t\t\t   WHEN -1 THEN DATALENGTH(st.text)\r\n\t\t\t\t   ELSE stats.statement_end_offset END - stats.statement_start_offset) / 2) + 1) AS query_text,\r\n\tISNULL(qp.query_plan, '') AS query_plan\r\nFROM qstats AS qs\r\n\t\tINNER JOIN sys.dm_exec_query_stats AS stats on qs.query_plan_handle = stats.plan_handle\r\n\t\tCROSS APPLY sys.dm_exec_query_plan(qs.query_plan_handle) AS qp\r\n\t\tCROSS APPLY sys.dm_exec_sql_text(qs.query_plan_handle) AS st;"
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1,33 +1,33 @@
        	            	-DECLARE @lookbackTime INT = -60;
        	            	-DECLARE @topNValue INT = 1000;
        	            	-with qstats as (
        	            	-	SELECT TOP(@topNValue)
        	            	-		REPLACE(@@SERVERNAME,'\',':') AS [sql_instance],
        	            	-		HOST_NAME() AS [computer_name],
        	            	-		MAX(qs.plan_handle) AS query_plan_handle,
        	            	-		qs.query_hash AS query_hash,
        	            	-		qs.query_plan_hash AS query_plan_hash,
        	            	-		SUM(qs.execution_count) AS execution_count,
        	            	-		SUM(qs.total_elapsed_time) AS total_elapsed_time,
        	            	-		SUM(qs.total_worker_time) AS total_worker_time,
        	            	-		SUM(qs.total_logical_reads) AS total_logical_reads,
        	            	-		SUM(qs.total_physical_reads) AS total_physical_reads,
        	            	-		SUM(qs.total_logical_writes) AS total_logical_writes,
        	            	-		SUM(qs.total_rows) AS total_rows,
        	            	-		SUM(qs.total_grant_kb) as total_grant_kb
        	            	-	FROM sys.dm_exec_query_stats AS qs
        	            	-	WHERE qs.last_execution_time BETWEEN DATEADD(SECOND, @lookbackTime, GETDATE()) AND GETDATE() 
        	            	-	GROUP BY
        	            	-		qs.query_hash,
        	            	-		qs.query_plan_hash
        	            	-)
        	            	-SELECT qs.*,
        	            	-	SUBSTRING(st.text, (stats.statement_start_offset / 2) + 1,
        	            	-			 ((CASE statement_end_offset
        	            	-				   WHEN -1 THEN DATALENGTH(st.text)
        	            	-				   ELSE stats.statement_end_offset END - stats.statement_start_offset) / 2) + 1) AS query_text,
        	            	-	ISNULL(qp.query_plan, '') AS query_plan
        	            	-FROM qstats AS qs
        	            	-		INNER JOIN sys.dm_exec_query_stats AS stats on qs.query_plan_handle = stats.plan_handle
        	            	-		CROSS APPLY sys.dm_exec_query_plan(qs.query_plan_handle) AS qp
        	            	+DECLARE @lookbackTime INT = -60;
        	            	+DECLARE @topNValue INT = 1000;
        	            	+with qstats as (
        	            	+	SELECT TOP(@topNValue)
        	            	+		REPLACE(@@SERVERNAME,'\',':') AS [sql_instance],
        	            	+		HOST_NAME() AS [computer_name],
        	            	+		MAX(qs.plan_handle) AS query_plan_handle,
        	            	+		qs.query_hash AS query_hash,
        	            	+		qs.query_plan_hash AS query_plan_hash,
        	            	+		SUM(qs.execution_count) AS execution_count,
        	            	+		SUM(qs.total_elapsed_time) AS total_elapsed_time,
        	            	+		SUM(qs.total_worker_time) AS total_worker_time,
        	            	+		SUM(qs.total_logical_reads) AS total_logical_reads,
        	            	+		SUM(qs.total_physical_reads) AS total_physical_reads,
        	            	+		SUM(qs.total_logical_writes) AS total_logical_writes,
        	            	+		SUM(qs.total_rows) AS total_rows,
        	            	+		SUM(qs.total_grant_kb) as total_grant_kb
        	            	+	FROM sys.dm_exec_query_stats AS qs
        	            	+	WHERE qs.last_execution_time BETWEEN DATEADD(SECOND, @lookbackTime, GETDATE()) AND GETDATE() 
        	            	+	GROUP BY
        	            	+		qs.query_hash,
        	            	+		qs.query_plan_hash
        	            	+)
        	            	+SELECT qs.*,
        	            	+	SUBSTRING(st.text, (stats.statement_start_offset / 2) + 1,
        	            	+			 ((CASE statement_end_offset
        	            	+				   WHEN -1 THEN DATALENGTH(st.text)
        	            	+				   ELSE stats.statement_end_offset END - stats.statement_start_offset) / 2) + 1) AS query_text,
        	            	+	ISNULL(qp.query_plan, '') AS query_plan
        	            	+FROM qstats AS qs
        	            	+		INNER JOIN sys.dm_exec_query_stats AS stats on qs.query_plan_handle = stats.plan_handle
        	            	+		CROSS APPLY sys.dm_exec_query_plan(qs.query_plan_handle) AS qp
        	            	 		CROSS APPLY sys.dm_exec_sql_text(qs.query_plan_handle) AS st;
        	Test:       	TestQueryTextAndPlanQueryContents/Test_query_text_and_query_plan_without_instance_name
```

--- a/receiver/sqlserverreceiver/queries_test.go
+++ b/receiver/sqlserverreceiver/queries_test.go
@@ -99,14 +99,12 @@ func TestQueryTextAndPlanQueryContents(t *testing.T) {
 
 	for _, tt := range queryTests {
 		t.Run(tt.name, func(t *testing.T) {
-			expectedBytes, err := os.ReadFile(path.Join("./testdata", tt.expectedQueryValFilename))
+			expected, err := os.ReadFile(path.Join("./testdata", tt.expectedQueryValFilename))
 			require.NoError(t, err)
-			// Replace all will fix newlines when testing on Windows
-			expected := strings.ReplaceAll(string(expectedBytes), "\r\n", "\n")
 
 			actual, err := tt.getQuery(tt.instanceName, tt.maxQuerySampleCount, tt.lookbackTime)
 			require.NoError(t, err)
-			require.Equal(t, strings.TrimSpace(expected), strings.TrimSpace(actual))
+			require.Equal(t, strings.TrimSpace(string(expected)), strings.TrimSpace(actual))
 		})
 	}
 }

